Control/move_to_pose/move_to_pose.py

```diff
@@ -173,16 +172,14 @@ def transformation_matrix(x, y, theta):
 def main():
 
     for i in range(5):
-        x_start = 20 * random()
-        y_start = 20 * random()
-        theta_start = 2 * np.pi * random() - np.pi
+        x_start: float = 20 * random()
+        y_start: float = 20 * random()
+        theta_start: float = 2 * np.pi * random() - np.pi
         x_goal = 20 * random()
         y_goal = 20 * random()
         theta_goal = 2 * np.pi * random() - np.pi
-        print("Initial x: %.2f m\nInitial y: %.2f m\nInitial theta: %.2f rad\n" %
-              (x_start, y_start, theta_start))
-        print("Goal x: %.2f m\nGoal y: %.2f m\nGoal theta: %.2f rad\n" %
-              (x_goal, y_goal, theta_goal))
+        print(f"Initial x: {round(x_start)} m\nInitial y: {round(y_start)} m\nInitial theta: {round(theta_start)} rad\n")
```

Comment:
These `round(x)` should be `round(x, 2)`?

Reply:
Yes. Will make the change